src/components/Dashboard/Capacity/CapacityItem.js

```diff
@@ -27,43 +24,63 @@ export const CapacityItem = ({ id, title, used, total, unit, formatValue, Loadin
-    used = usedFormatted.value;
+    const formattedUnit = totalFormatted.unit; // eslint-disable-line prefer-destructuring
+    const formattedTotal = totalFormatted.value;
+    const formattedUsed = usedFormatted.value;
 
-    const available = total - used;
-    columns = [[USED, used], [AVAILABLE, available]];
+    const available = formattedTotal - formattedUsed;
 
-    const percentageUsed = total > 0 ? Math.round((100 * used) / total) : 0;
+    const percentageUsed = total > 0 ? Math.round((100 * formattedUsed) / formattedTotal) : 0;
     const availableNice = Number.isInteger(available) ? available : available.toFixed(1);
-    const totalNice = Number.isInteger(total) ? total : total.toFixed(1);
+    const totalNice = Number.isInteger(formattedTotal) ? formattedTotal : formattedTotal.toFixed(1);
+    const usedNice = Number.isInteger(formattedUsed) ? formattedUsed : formattedUsed.toFixed(1);
 
-    description = `${availableNice} ${unit} available out of ${totalNice} ${unit}`;
-    tooltip = { contents: pfGetUtilizationDonutTooltipContents };
+    data = [
+      {
+        x: USED,
+        y: formattedUsed,
+        label: `${USED}: ${usedNice}${formattedUnit}`,
+      },
+      {
+        x: AVAILABLE,
+        y: available,
+        label: `${AVAILABLE}: ${availableNice}${formattedUnit}`,
+      },
+    ];
+    description = `${availableNice} ${formattedUnit} available out of ${totalNice} ${formattedUnit}`;
     primaryTitle = `${percentageUsed.toString()}%`;
     secondaryTitle = USED;
   }
-  // TODO(mlibra): align text font
   return (
     <div className="kubevirt-capacity__item">
       <h3 className="kubevirt-capacity__item-title">{title}</h3>
       <h6>{description}</h6>
-      <div>
-        <DonutChart
-          id={prefixedId('donut-chart', id)}
-          size={{ width: 150, height: 150 }}
-          data={{
-            unload: true,
-            columns,
-            order: null,
-          }}
-          unloadBeforeLoad
-          tooltip={tooltip}
-          title={{
-            primary: primaryTitle,
-            secondary: secondaryTitle,
-          }}
-        />
+      <div id={prefixedId('donut-chart', id)} className="kubevirt-capacity__graph">
+        <svg viewBox={`0 0 ${CHART_WIDTH} ${CHART_HEIGHT}`}>
+          <ChartDonut
+            data={data}
+            labels={datum => datum.label}
+            theme={ChartTheme.light.blue}
+            height={CHART_HEIGHT}
+            width={CHART_WIDTH}
+            standalone={false}
+            labelComponent={<ChartTooltip style={{ fontSize: 20 }} />}
+          />
+          <ChartLabel
+            textAnchor="middle"
+            style={{ fontSize: 25 }}
+            x={CHART_WIDTH / 2}
+            y={CHART_HEIGHT / 2.3}
+            text={primaryTitle}
+          />
+          <ChartLabel
+            textAnchor="middle"
+            style={{ fontSize: 15 }}
+            x={CHART_WIDTH / 2}
+            y={CHART_HEIGHT / 1.8}
```

Comment:
shouldn't this reather be `CHART_HEIGHT / 2 - SOME_CONSTANT` ? and the `x` one also?